ui.apps/src/main/content/jcr_root/apps/training/components/content/registration/clientlibs/js/registration.js

```diff
@@ -15,17 +14,29 @@ $('#submit').click(function() {
 
     document.body.append(hobbies);
 
+
+    $(".error").remove();
+
+    if (firstName.length < 1) {
+      $('#firstname').after('<span class="error">This field is required</span>');
+    }
+    else if (lastName.length < 1) {
+      $('#lastname').after('<span class="error">This field is required</span>');
+    }
+    else{
+
     //Use JQuery AJAX request to post data to a Sling Servlet
     $.ajax({
          type: 'POST',
-         url:'/bin/registerServlet',
+         url:'/bin/registerTrainingServlet',
          data:'firstName='+ firstName+'&lastName='+ lastName+'&hobbies='+ hobbies,
          success: function(msg){
 
 
```

Comment:
don't use alert and also pls format the code before pushing changes

Reply:
fixed